contactsummary/js/src/contact-summary/controllers/keyDetails.js

```diff
@@ -31,8 +31,13 @@ define([
                 }
 
                 this.primaryContract = response;
-                this.primaryContract.lengthOfService = getLengthOfService(response.start_date, response.end_date);
-            }.bind(this))
+            }.bind(this)).then(function (response) {
+                return Contract.getLengthOfServiceYmd();
```

Comment:
we're following a 2 space indentation convention now. Use it for all of your new code and when possible (and when reasonable) amend the old code